pandas/io/json/_normalize.py

```diff
@@ -469,7 +469,16 @@ def _recursive_extract(data, path, seen_meta, level=0):
             for obj in data:
                 for val, key in zip(_meta, meta_keys):
                     if level + 1 == len(val):
-                        seen_meta[key] = _pull_field(obj, val[-1])
+                        try:
+                            seen_meta[key] = _pull_field(obj, val[-1])
+                        except KeyError as e:
+                            if errors == "ignore":
+                                seen_meta[key] = np.nan
+                            else:
+                                raise KeyError(
+                                    "Try running with errors='ignore' as key "
```

Comment:
why the custom message (I get that's it helpful), but this is a bit non-standard.

you can do something like

 f`{key} not Found. Pass errors='ignore' to ignore not found keys` (or similar )